pylint_plugins/assert_raises_without_msg.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import astroid
+from pylint.interfaces import IAstroidChecker
+from pylint.checkers import BaseChecker
+
+
+def _is_assert_raises_without_msg(node: astroid.Call):
+    return (
+        isinstance(node.func, astroid.Attribute)
+        and node.func.as_string() == "self.assertRaises"
+        and "msg" not in [k.arg for k in node.keywords]
+    )
+
+
+IGNORE_FILES = list(
+    map(
+        os.path.abspath,
+        [
+            # Instructions
+            # ============
+            # 1. Select a file in the list below and remove it.
+            # 2. Run pylint and confirm it fails.
+            # 3. Fix the lines printed out in the previous step.
+            # 4. Run pylint again and confirm it succeeds now.
+            # 5. Run pytest and confirm the changed lines don't fail.
+            # 6. Open a PR.
```

Comment:
I'm going to create an issue ticket for this and let contributors fix the listed files.

Reply:
good idea! Let's get @erich-db on this one